frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -68,3 +95,21 @@ export const EMPTY_AWS_SECRET_DATA: AWSDataEntry = [
     value: '',
   },
 ];
+export const EMPTY_AWS_PIPELINE_DATA: AWSDataEntry = [
```

Comment:
Can't we reuse `EMPTY_AWS_SECRET_DATA`? Again this is a subset of `EMPTY_AWS_SECRET_DATA`.

Reply:
To simplify the things are have separated them